app/_hub/kong-inc/request-validator/index.md

```diff
@@ -32,14 +31,16 @@ params:
   consumer_id: true
   config:
     - name: body_schema
-      required: true
+      required: false
       value_in_examples: '''[{"name":{"type": "string", "required": true}}]'''
-      description: Array of schema fields.
+      datatype: string
+      description: The request body schema specification.
 
     - name: allowed_content_types
-      required: false
+      required: true
```

Comment:
this is not required

at least one of these fields must be non-empty: 'body_schema', 'parameter_schema'

```bash
# http :8001/plugins name=request-validator
HTTP/1.1 400 Bad Request
Access-Control-Allow-Origin: *
Connection: keep-alive
Content-Length: 295
Content-Type: application/json; charset=utf-8
Date: Wed, 03 Mar 2021 14:18:06 GMT
Server: kong/2.2.0.0-enterprise-edition
X-Kong-Admin-Latency: 1404
X-Kong-Admin-Request-ID: 7Su9JpIpN3V4hEvv0xilLQNCNzS5lZHA
vary: Origin

{
    "code": 2,
    "fields": {
        "config": {
            "@entity": [
                "at least one of these fields must be non-empty: 'body_schema', 'parameter_schema'"
            ]
        }
    },
    "message": "schema violation (config.@entity: {\n  \"at least one of these fields must be non-empty: 'body_schema', 'parameter_schema'\"\n})",
    "name": "schema violation"
}
```

see this constraint on the schema.lua: https://github.com/Kong/kong-plugin-enterprise-request-validator/blob/master/kong/plugins/request-validator/schema.lua#L142-L143

so how do we communicate this?

Reply:
We'll handle it like other plugins with "semi" and more info in description 
![image](https://user-images.githubusercontent.com/3756245/109848651-1f3abd00-7c16-11eb-9c5c-f41d9e9a4433.png)